src/PaymentGateways/Gateways/TestOffsiteGateway/testOffsiteGateway.tsx

```diff
@@ -0,0 +1,20 @@
+import {Gateway} from '@givewp/forms/types';
+
+interface TestOffsiteGateway extends Gateway {
+    settings?: {
+        label: string;
+        message: string;
+    };
+}
+
+/**
+ * @unreleased
+ */
+const testOffsiteGateway: TestOffsiteGateway = {
+    id: 'test-offsite-gateway',
+    Fields() {
```

Comment:
When I try to compile the assets I got the following error on this line:

```
 TS2322: Type '() => string' is not assignable to type 'FC<{}>'.
  Type 'string' is not assignable to type 'ReactElement<any, any>'.
```

![image](https://github.com/impress-org/givewp/assets/16308864/5f54be1c-c97f-4df4-9594-89849119cc29)

Reply:
Resolved 4a7dec09